### ISSUE TITLE
Is there a proper step-by-step instruction?

### ISSUE THREAD

**Issue by r0oland** [platformio/bintray-secure-ota#10]
I find the video hard to follow, and it doesn't really help me understand what is happening. Is there a proper step-by-step instruction that a script kiddy like me can follow?

**Comment by cyberman54** [comment]
no.

**Comment by r0oland** [comment]
See my pull-request:
https://github.com/platformio/bintray-secure-ota/pull/14

**Comment by r0oland** [comment]
Any comments on my pull request?

**Comment by r0oland** [comment]
Well, if anyone needs step-by-step instructions, check out my fork:
https://github.com/r0oland/bintray-secure-ota
